src/Entities/Pawn.cpp

```diff
@@ -70,7 +70,7 @@ void cPawn::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 			Vector3d v3Delta = a_Entity->GetPosition() - m_Pusher->GetPosition();
 			v3Delta.y = 0.0;  // we only push sideways
-			v3Delta *= 1.0 / v3Delta.Length();  // we push harder if we're close
+			v3Delta *= 1.0 / (v3Delta.Length() + 0.01);  // we push harder if we're close
```

Comment:
A square root is always non-negative. I think this is the proper way to fix the issue.